modules/objdetect/src/qrcode.cpp

```diff
@@ -2251,6 +2252,53 @@ bool QRDecode::preparingCurvedQRCodes()
     return true;
 }
 
+/**
+ * @param vertices - 12 points of alignment markers
+ * @param original - 4 points of QR code
+ * @return pair<int, int> - vertex index in vertices, vertex index in original
+ */
+static inline std::pair<int, int> getVertIndexes(const vector<Point> &vertices, const vector<Point2f> original) {
```

Comment:
The name does not contain function semantic. I propose something like `nearestVertices` or `closestPair`.